index.js

```diff
@@ -74,7 +78,7 @@ module.exports = opts => {
 		}
 	});
 
-	if (Object.keys(cli).length) {
+	if (Object.keys(cli).length >== 0) {
```

Comment:
This was one of the linting errors that were showing on travis. Should I leave it as it was?